.github/workflows/storybook.yaml

```diff
@@ -0,0 +1,51 @@
+name: Deploy Storybook to Cloudflare Pages
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  deployment:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        name: Check out the repository
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18'
```

Comment:
```suggestion
```

do not specify node version since we have declared in `.node-version` file https://github.com/the-guild-org/docs/blob/ac178a3d87ff62fff42fc98271f2bc74ea816440/.node-version